File: samples/java/src/main/java/co/windly/limbosample/presentation/main/root/RootView.java
package co.windly.limbosample.presentation.main.root;

import co.windly.limbo.mvp.fragment.base.LimboFragmentView;

public interface RootView extends LimboFragmentView {

  //region Navigation

  void navigateToHomeView();

  void navigateToSettingsView();

  //endregion
}

File: samples/kotlin/src/main/java/co/windly/limbosample/presentation/main/root/RootView.kt
package co.windly.limbosample.presentation.main.root

import co.windly.limbo.fragment.base.LimboFragmentView

interface RootView : LimboFragmentView {

  //region Navigation

  fun navigateToHomeView()

  fun navigateToSettingsView()

  //endregion
}
